exporter/clickhousetracesexporter/writer.go

```diff
@@ -199,7 +200,20 @@ func (w *SpanWriter) writeIndexBatch(batchSpans []*Span) error {
 		}
 	}
 
-	return statement.Send()
+	start := time.Now()
+
+	err = statement.Send()
+	if err != nil {
+		return err
+	}
+
+	ctx, _ = tag.New(ctx,
```

Comment:
As suggested by Srikanth, we should capture the metric even if an error occurred.

Reply:
ok